wai-aria/scripts/aria-utils.js

```diff
@@ -136,10 +136,17 @@ const AriaUtils = {
       let label = el.getAttribute("data-expectedlabel");
       let testName = el.getAttribute("data-testname") || label; // data-testname optional if label is unique per test file
       promise_test(async t => {
-        const expectedLabel = el.getAttribute("data-expectedlabel");
+        let expectedLabel = el.getAttribute("data-expectedlabel");
         let computedLabel = await test_driver.get_computed_label(el);
-        // Todo: Remove whitespace normalization after https://github.com/w3c/accname/issues/192 is addressed. Change prior line back to `const`, too.
-        computedLabel = computedLabel.trim()
+
+        // See: https://github.com/w3c/accname/issues/192 and https://github.com/w3c/accname/issues/208
+        //
+        // HTML/JS whitespace matches regular space, formfeed (\f), newline (\n), return (\r), tab (\t), vertical tab (\v),
+        // non-breaking space (\xA0), and a few more Unicode whitespace characters: \u00A0, \u2028, and \u2029.
+        const whitespace = /\s+/g;
```

Comment:
https://github.com/w3c/accname/pull/165 resolved to use https://infra.spec.whatwg.org/#ascii-whitespace (but seemingly only for the `aria-label` attribute). `\s` matches more characters: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Guide/Regular_expressions/Cheatsheet#character_classes

Reply:
Thanks. Updated. 
https://github.com/web-platform-tests/wpt/pull/42407/files